src/util/electionDefinition.ts

```diff
@@ -31,9 +27,9 @@ export type ValidationError =
 export function* validate(
   electionDefinition: ElectionDefinition
 ): Generator<ValidationError> {
-  const actualJSON = electionDefinition.electionData
+  const actualJSON = JSON.stringify(JSON.parse(electionDefinition.electionData))
```

Comment:
Round-trip check.

Reply:
I think this is effectively adding canonicalization --> the original `electionData` better be the canonical serialization. Are we sure we want this?

Reply:
This just means that the way you serialize `election` has to match the way `electionData` round trips. Can you come up with an example where this is a problem?